code/procs/gamehelpers.dm

```diff
@@ -776,3 +776,84 @@ proc/get_ouija_word_list(var/atom/movable/source = null, var/words_min = 5, var/
 					words |= (M.real_name ? M.real_name : M.name)
 
 	return words
+
+
+
+// basically the same as the ouija board, but more said_phrases
+proc/get_plush_word_list(var/atom/movable/source = null, var/words_min = 5, var/words_max = 8, var/include_nearby_mobs_chance = 40, var/include_most_mobs_chance = 20, include_said_phrases_chance = 25)
+	var/list/words = list()
+
+	// Generic words
+	for(var/i in 1 to rand(words_min, words_max))
+		var/picked = pick(strings("plush_toy_words.txt", "plush_toy_words"))
+		words |= picked
+
+	if (prob(include_nearby_mobs_chance))
+		var/list/mobs = observersviewers(Center = source)
+		if (length(mobs))
+			var/mob/M = pick(mobs)
+			words |= (M.real_name ? M.real_name : M.name)
+
+	if(prob(include_said_phrases_chance))
+		words |= phrase_log.random_phrase("say")
+
+	if (prob(include_most_mobs_chance))
+
+		var/roll = rand(0, 200)
+		switch (roll)
+			if (0)
+				// any actual antag
+				var/list/player_pool = list()
+				for (var/mob/M in mobs)
+					if (!M.client || istype(M, /mob/new_player) || !checktraitor(M))
+						continue
+					player_pool += M
+				if (length(player_pool))
+					var/mob/M = pick(player_pool)
+					words |= (M.real_name ? M.real_name : M.name)
+			if (1 to 5)
+				// fake wraith
+				words |= call(/mob/wraith/proc/make_name)()
+			if (6 to 10)
+				// fake blob (heh)
+				var/blobname = phrase_log.random_phrase("name-blob")
+				words |= strip_html(copytext(blobname, 1, 26) + " the Blob")
+			if (10 to 20)
+				// fake nukeop (uses the real nukeop company name, too)
+				// Copied from gamemodes/nuclear.dm
+				var/list/callsign_pool_keys = list("nato", "melee_weapons", "colors", "birds", "mammals", "moons")
+				//Alphabetical agent callsign lists are delcared here, seperated in to catagories.
+				var/list/callsign_list = strings("agent_callsigns.txt", pick(callsign_pool_keys))
+				words |= "[syndicate_name()] Operative [pick(callsign_list)]"
+			if (20 to 30)
+				// fake wizard
+				var/wizname = phrase_log.random_phrase("name-wizard")
+				words |= strip_html(copytext(wizname, 1, 26))
+
+			else
+				// any random living mob
+				var/list/player_pool = list()
+				for (var/mob/M in mobs)
+					if (!M.client || istype(M, /mob/new_player))
+						continue
+					player_pool += M
+				if (length(player_pool))
+					var/mob/M = pick(player_pool)
+					words |= (M.real_name ? M.real_name : M.name)
+
+	return words
+
+
+=======
+// returns initial health of an item or an item type
+/proc/get_initial_item_health(obj/item/I)
+	if (initial(I.health))
+		return initial(I.health)
+	else
+		var/weight_class = initial(I.w_class)
+		switch (weight_class)
+			if (W_CLASS_TINY to W_CLASS_NORMAL)
+				return weight_class + 1
+			else
+				return weight_class + 2
+>>>>>>> master
```

Comment:
These bits are git merge markers, they're meant to be removed when you accept changes. Probably best to look up a guide on how to resolve git merge conflicts or feel free to ask in the goonstation discord